packages/mdc-dialog/README.md

```diff
@@ -304,9 +308,9 @@ Opens the dialog, registers appropriate event listeners, sets aria attributes, f
 Closes the dialog, deregisters appropriate event listeners, resets aria attributes, focuses
 elements.
 
-#### MDCDialogFoundation.accept(notifyChange = false) => void
+#### MDCDialogFoundation.yes(notifyChange = false) => void
```

Comment:
Some clients will probably need to handle <kbd>Enter</kbd> key presses for custom XHR forms

Reply:
If we go with not hard-coding these buttons,  these APIs won't make sense anymore anyway.

My first instinct is to say that handling binding of form enter keypresses to buttons is a concern of whoever implements the content in the dialog.

My second instinct, which is probably a stretch and we shouldn't open the can of worms right now, is to maybe support `<form>` in the dialog DOM somewhere, so that a submit button will Just Work (and potentially so will validation?)

Reply:
> form enter keypresses to buttons is a concern of whoever implements the content in the dialog

👍